app/scripts/modules/core/src/loadBalancer/LoadBalancersTag.spec.tsx

```diff
@@ -63,17 +65,27 @@ describe('<LoadBalancersTag />', () => {
     application.getDataSource('loadBalancers').data = [ lb1, lb2 ];
 
     const props: ILoadBalancersTagProps = { application, serverGroup };
-    component = mount(<LoadBalancersTag {...props}/>);
+    const popoverContainerEl = document.createElement('div');
+    component = mount(<LoadBalancersTag {...props} container={popoverContainerEl} />);
```

Comment:
@anotherchrisberry this is what I ended up with.  Because the popover code invokes `React.render()` I couldn't use enzyme to test the popover portion, but react itself updates the raw DOM element and that's good enough for this test.